### PR TITLE
2.10.0 Changelog

### PR DESCRIPTION
**Pull request by enrico-usai**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,15 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 
-- Add support for CentOS 8 in all Commercial and GovCloud regions.
+- Add support for CentOS 8 in all Commercial regions.
+- Add support for instance types with multiple network cards like P4d.
 - Enable FSx Lustre on China regions.
```
Comment (demartinofra) [inline CHANGELOG.md:13]: in China regions

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,12 +8,15 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 
-- Add support for CentOS 8 in all Commercial and GovCloud regions.
+- Add support for CentOS 8 in all Commercial regions.
+- Add support for instance types with multiple network cards like P4d.
```
Comment (demartinofra) [inline CHANGELOG.md:12]: I would remove like P4d and at most replace it with `(e.g. p4d.24xlarge)

File: CHANGELOG.md [hunk 3]
```diff
@@ -25,15 +28,24 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   - Introduces ``-g/--enable-gdr`` switch to install packages with GPUDirect RDMA support
   - Updates to OMPI collectives decision file packaging, migrated from efa-config to efa-profile
   - Introduces CentOS 8 support
-  
 - CentOS 6 is no longer supported.
-- Install python3 version of ``aws-cfn-bootstrap`` scripts
+- Upgrade NVIDIA driver to version 450.80.02.
+- Install NVIDIA Fabric manager to enable NVIDIA NVSwitch on supported platforms.
```
Comment (demartinofra) [inline CHANGELOG.md:33]: isn't this an enhancement?

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,12 +8,15 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 
-- Add support for CentOS 8 in all Commercial and GovCloud regions.
+- Add support for CentOS 8 in all Commercial regions.
+- Add support for instance types with multiple network cards like P4d.
 - Enable FSx Lustre on China regions.
-- Add support for instance types with multiple network cards like P4d
+- Add validation step for AMI creation process to fail when using an AMI created by other versions of ParallelCluster
```
Comment (demartinofra) [inline CHANGELOG.md:14]: Fail AMI creation process when using...

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,12 +8,15 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
+  as base AMI.
+- Add validation step for AMI creation process to fail if the selected OS and the base AMI OS are not consistent.
```
Comment (demartinofra) [inline CHANGELOG.md:16]: same here

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,12 +8,15 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 
-- Add support for CentOS 8 in all Commercial and GovCloud regions.
+- Add support for CentOS 8 in all Commercial regions.
```
Comment (demartinofra) [inline CHANGELOG.md:11]: probably in cookbook we don't need to mention the regions

File: CHANGELOG.md [hunk 7]
```diff
@@ -25,15 +28,24 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
 - Retrieve FSx Lustre DNS name dynamically.
+- Slurm: change `SlurmctldPort` to 6820-6829 to not overlap with default `slurmdbd` port (6819).
+- Slurm: add `compute_resource` name and `efa` as node features.
+- Improve Slurm and Munge installation process by cleaning up existing installations from OS repositories.
+- Install Python 3 version of ``aws-cfn-bootstrap`` scripts.
+- Do not force compute fleet into `STOPPED` state when performing a cluster update. This allows to update the queue
+  size without forcing a termination of the existing instances.
+
+**BUG FIXES**
+- Improve ephemeral drives setup to avoid failures when partition changes require a reboot.
```
Comment (demartinofra) [inline CHANGELOG.md:45]: nit: I'd not use the Improve word in bug fixes.

**Review by demartinofra — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,12 +8,16 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 
-- Add support for CentOS 8 in all Commercial and GovCloud regions.
-- Enable FSx Lustre on China regions.
-- Add support for instance types with multiple network cards like P4d
+- Add support for CentOS 8.
+- Add support for instance types with multiple network cards (e.g. `p4d.24xlarge`).
+- Enable FSx Lustre in China regions.
+- Fail AMI creation process when using an AMI created by other versions of ParallelCluster
```
Comment (mauri-melato) [inline CHANGELOG.md:14]: This was reworded in the main change log

File: CHANGELOG.md [hunk 9]
```diff
@@ -8,12 +8,16 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
+  as base AMI.
+- Fail AMI creation process when using an OS not consinstent with the one in the base AMI.
```
Comment (mauri-melato) [inline CHANGELOG.md:16]: consistent

File: CHANGELOG.md [hunk 10]
```diff
@@ -8,12 +8,16 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
+- Add possibility to use a post installation script when building an AMI.
+- Install NVIDIA Fabric manager to enable NVIDIA NVSwitch on supported platforms.
```
Comment (mauri-melato) [inline CHANGELOG.md:18]: This is a CHANGE to be reported in the main changelog

File: CHANGELOG.md [hunk 11]
```diff
@@ -25,15 +29,23 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
-- Install python3 version of ``aws-cfn-bootstrap`` scripts
+- Upgrade NVIDIA driver to version 450.80.02.
 - Upgrade Intel Parallel Studio XE Runtime to version 2020.2.
-- Do not force compute fleet into STOPPED state when performing a cluster update. This allows to update the queue
-  size without forcing a termination of the existing instances.
-- Upgrade NVIDIA driver to version 450.80.02
-- Install NVIDIA Fabric manager to enable NVIDIA NVSwitch on supported platforms
+- Upgrade Munge to version 0.5.14.
 - Retrieve FSx Lustre DNS name dynamically.
+- Slurm: change `SlurmctldPort` to 6820-6829 to not overlap with default `slurmdbd` port (6819).
```
Comment (mauri-melato) [inline CHANGELOG.md:37]: To be reported in the main changelog

File: CHANGELOG.md [hunk 12]
```diff
@@ -25,15 +29,23 @@ This file is used to list changes made in each version of the AWS ParallelCluste
...
+- Slurm: add `compute_resource` name and `efa` as node features.
```
Comment (mauri-melato) [inline CHANGELOG.md:38]: To be reported in the main changelog

**Review by demartinofra — APPROVED** [review]

**Pull request merged**